pkgs/development/tools/qtcreator/default.nix

```diff
@@ -6,8 +6,8 @@
 with stdenv.lib;
 
 let
-  baseVersion = "4.6";
```

Comment:
It might make sense to do what GNOME packages do: define a single version attribute and split it where needed. It will allow us to us `update-source-version` program.

https://github.com/NixOS/nixpkgs/blob/14850326865cda14d314c85942851c30774916f9/pkgs/development/libraries/libdazzle/default.nix#L15